src/app/api/services/dot-starter/dot-starter.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from 'dotcms-js';
+
+@Injectable()
+export class DotStarterService {
```

Comment:
```suggestion
export class DotToolGroupStarterService {
```

Right now it looks like a service to handle "starters" and is not that.

Don't forget file names.